src/components/integrated/OakQuizRadioButton/OakQuizRadioButton.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { MouseEventHandler, useContext } from "react";
+import styled, { css } from "styled-components";
+
+import { OakBox, OakFlex, OakIcon } from "@/components/base";
+import {
+  OakRadioButton,
+  OakRadioButtonProps,
+} from "@/components/ui/OakRadioButton/OakRadioButton";
+import { parseDropShadow } from "@/styles/helpers/parseDropShadow";
+import { parseBorder } from "@/styles/helpers/parseBorder";
+import { parseColor } from "@/styles/helpers/parseColor";
+import { parseBorderRadius } from "@/styles/helpers/parseBorderRadius";
+import { RadioContext } from "@/components/ui/OakRadioGroup/OakRadioGroup";
+import { OakUiRoleToken } from "@/styles";
+
+export type OakQuizRadioButtonProps = OakRadioButtonProps & {
+  feedback?: "correct" | "incorrect" | null;
+  image?: JSX.Element;
+};
+
+type StyledOakFlexProps = {
+  $disabled: boolean;
+  $checked: boolean;
+  $outlineColor?: OakUiRoleToken;
+};
+
+const StyledOakFlex = styled(OakFlex)<StyledOakFlexProps>`
+  ${(props) =>
+    !!props.$outlineColor &&
+    css`
+      outline: ${parseBorder("border-solid-xl")}
```

Comment:
Using `outline` to perfectly match the design where the border is rendered outside the element (having no impact on its internal dimensions) meaning that the full 8px of rounding is applied internally.

<img width="316" alt="Screenshot 2024-01-10 at 15 04 47" src="https://github.com/oaknational/oak-components/assets/122096/95f2ab7d-f75b-4048-8a69-e2103bbe72c2">